README.md

```diff
@@ -103,105 +113,170 @@ The user microservice manages the lifecycle of users and user login within the A
 * To allow users to login using Twitter, complete the [Twitter Configuration](#twitter-configuration)
 
 #### Build and Start
+##### Maven
 Issue the following commands from within sample-acmegifts/microservice-user directory
 
     mvn clean package install
     mvn exec:exec@start-database
     mvn liberty:start-server
     
+##### Gradle
+Issue the following commands from within sample-acmegifts/microservice-user directory
+
+	gradle clean build
+	gradle libertyStart
+
+Calling `libertyStart` should automatically trigger `startDatabase`.
```

Comment:
"should" sounds like you are not sure.  can you say "will"?  Are there scenarios this doesn't work?

Reply:
you're right. bad habit. fixed to "automatically triggers"